ktor-utils/common/src/io/ktor/util/PlatformUtils.kt

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.util
+
+@InternalAPI
+expect object PlatformUtils {
```

Comment:
This is definitely a thing that everyone would abuse :skull_and_crossbones:

Reply:
Probably, but we also can't live without it :)